base/applications/notepad/lang/tr-TR.rc

```diff
@@ -1,5 +1,6 @@
 // COPYRIGHT: 2012 Arda Tanrıkulu <ardatanrikulu@gmail.com>
 //            2013-2019 Erdem Ersoy
+//            2024-Mahir GÜL mahirgul@gmail.com
```

Comment:
Please use the last name without Caps Lock (only first letter is supposed to be capitalized) - and please apply the same to your GitHub profile.
```suggestion
//            2024 Mahir Gül <mahirgul@gmail.com>
```